README.md

```diff
@@ -1,97 +1,37 @@
-
-![lighthouse report](../assets/images/lighthouse-report.png)
-
-### [Webhint](https://webhint.io/)
-
-A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
-
-**NOTE:** If you are running on Windows, you need to initialize npm to create `package.json` file. 
-   ```
-   npm init -y
-   ```
-
-1. Run
-   ```
-   npm install --save-dev hint@7.x
-   ```
-   *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
-2. Copy [.hintrc](.hintrc) to the root directory of your project.
-3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-4. Run
-   ```
-   npx hint .
-   ```
-5. Fix validation errors.
-
-### [Stylelint](https://stylelint.io/)
-
-A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
-
-1. Run
-
-   ```
-   npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-   ```
-
-   *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
-
-2. Copy [.stylelintrc.json](./.stylelintrc.json) to the root directory of your project.
-3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-4. Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
-5. Fix linter errors.
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/cli#autofixing-errors) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
-
-## Have you already completed the Sass lessons?
-
-If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
-a few modifications to the linter configuration files.
-
-1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
-2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
-
-Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
-every time.
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] @RodChak as previously requested kindly add `MIT.md` file or remove the license altogether from your readme file.

Reply:
Hey @danmainah, thank you for the feedback, really appreciate it. Could you please assist me with this part, I've already made 3 changes thinking I'm doing the correct thing, but it seems I'm not doing it correctly. I've added this [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) license file in my root directory, but my third reviewer still insisted that I correct that part, could you please assist me if you have some time, I've attached a screenshot of how I did it in my README.md file.
![Screenshot 2022-06-29 1](https://user-images.githubusercontent.com/97179349/176546261-06435dff-eb9b-46f8-bb01-9917922282ba.jpg)